packages/map-template/README.md

```diff
@@ -1,88 +1,114 @@
 # Map Template
 
-This app is built on React, with our Web Components used where applicable.
+## Web component: Installation and usage
 
-## Running the app
+### Using NPM
```

Comment:
@ammapspeople I made some changes here (and further down the document), as you're not allowed to have multiple H1 headings in a Markdown document.